spec/unit/dry/schema/dsl_spec.rb

```diff
@@ -34,4 +34,32 @@
       expect(rules[:user].(user: { name: '' })).to be_failure
     end
   end
+
+  describe '#config' do
+    let(:namespace) { :test_namespace }
+    let(:replacement_namespace) { :replacement_test_namespace }
+
+    it "uses a dup of the parent's config when a parent is present" do
+      config = Dry::Schema::Config.new.tap { |c| c.messages.namespace = namespace }
+      parent = Dry::Schema::DSL.new(config: config)
+      dsl = Dry::Schema::DSL.new(parent: parent)
+
+      aggregate_failures do
```

Comment:
all failures are aggregated automatically, there's a way to do it
https://github.com/dry-rb/dry-schema/blob/ab17647d5fe62b22ff1edeccf9a934c4eadc9e7d/spec/spec_helper.rb#L64-L66

Reply:
Thanks, you learn something new every day :)